azure/postgresql/ServiceApprovalAcceleratorPostgreSQL.md

```diff
@@ -0,0 +1,416 @@
+# Overview
+
+This section is meant to provide an opinionated approach towards the implementation security controls by domain. Although the approaches may not be a fit for all use cases or complete for production use,  
+they are meant to guide the audience towards current best practices and design considerations to achieve security control objectives.
+
+Azure database for PostgreSQL is available in 2 deployment options.
+- Single Server
+- Hyperscale (Citus) – scale horizontally across multiple nodes
+
+Although this document largely applies to both the offerings, some of the sections have been created with single server as the focus.
+
+## Controls and Architectures
+This table maps Security Domain to the corresponding controls and architectural best practices as documented in Azure public documentation, white papers, and blog posts.
+
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-7btt">Top level</th>
+    <th class="tg-fymr">Breakdown</th>
+    <th class="tg-fymr">Detail</th>
+    <th class="tg-fymr">Public Links</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-amwm">External Certification</td>
+    <td class="tg-8zwo"><a href="https://azure.microsoft.com/fr-fr/blog/compliance-offerings-for-azure-database-for-mysql-and-azure-database-for-postgresql/">Article on Compliance</a></td>
+    <td class="tg-0lax">
+        <p>Azure Database for PostgreSQL is certified compliant for:</p>
+        <ul>
+            <li>ISO 27001:2013</li>
+            <li>ISO 27018:2014*</li>
+            <li>CSA STAR Attestation</li>
+            <li>CSA STAR Certification</li>
+            <li>HIPAA / HITECH Act</li>
+            <li>PCI DSS Level 1</li>
+            <li>SOC1</li>
+            <li>SOC2</li>
+            <li>SOC3</li>
+            <li>EU Model Clauses</li>   
+            <li>UK G-cloud</li>
+        </ul>
+    </td>
+    <td class="tg-0lax"><a href="https://azure.microsoft.com/fr-fr/blog/compliance-offerings-for-azure-database-for-mysql-and-azure-database-for-postgresql/">Article on Compliance</a></td>
+  </tr>
+  <tr>
+    <td class="tg-7btt" rowspan="3">Identity &amp; Access Management</td>
+    <td class="tg-fymr">Authentication</td>
+    <td class="tg-f8tv">
+        <p>
+            It is recommended to enable AAD integration with PAzure Database for PostgreSQL.
+            <br><br>
+            Microsoft Azure Active Directory (Azure AD) authentication is a mechanism of connecting to Azure Database for PostgreSQL using identities defined in Azure AD.
+            With Azure AD authentication, you can manage database user identities and other Microsoft services in a central location, which simplifies permission management.            
+            <br><br>
+            when using Azure AD for authentication, the security features that build in AAD can help to protect the login process. The AAD token can be obtained in various
+            ways which could help to protect most of the password attach. 
```

Comment:
Is there a typo here? "Protect most of the password attach"

Reply:
+1